dev/org.eclipse.codewind.openapi.ui/src/org/eclipse/codewind/openapi/ui/Messages.properties

```diff
@@ -48,7 +48,14 @@ WIZARD_PAGE_SELECT_GENERATOR_TYPE=Select a generator type.
 BROWSE_DIALOG_MESSAGE_SELECT_DEFINITION=Select an OpenAPI definition from the project.
 BROWSE_DIALOG_TITLE_SELECT_DEFINITION=Select an OpenAPI definition file.
 
-BROWSE_DIALOG_MESSAGE_SELECT_FOLDER=Select an output folder for the generated files.
-BROWSE_DIALOG_TITLE_SELECT_FOLDER=Select a folder.
+BROWSE_DIALOG_MESSAGE_SELECT_FOLDER=Select an output folder under the current project
+BROWSE_DIALOG_OUTPUT_FOLDER_INVALID=Output Folder Invalid
+BROWSE_DIALOG_OUTPUT_FOLDER_INVALID_MESSAGE=The selected folder must be the project root folder or one of project's child folders.
```

Comment:
- This statement might be easier to grasp as a command...something like, "Make the selected folder the project root folder or one of project's child folders."

Reply:
Is this ok?  "Select an output folder that is either the project root folder or one of the project's child folders."

Reply:
Sounds good!